robot.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import wpilib
+import wpilib.event
+import magicbot
+
+from components.chassis import Chassis
+from controllers.movement import Movement
+
+from utilities.scalers import rescale_js
+
+
+class MyRobot(magicbot.MagicRobot):
+    # Components
+    chassis: Chassis
+
+    movement: Movement
+
+    max_speed = magicbot.tunable(Chassis.max_wheel_speed * 0.95)
+
+    def createObjects(self) -> None:
+        self.data_log = wpilib.DataLogManager.getLog()
+
+        self.gamepad = wpilib.XboxController(0)
+        self.joystick = wpilib.Joystick(1)
+
+        self.event_loop = wpilib.event.EventLoop()
+        # Right trigger events
+        self.right_trigger_down_full = self.gamepad.rightTrigger(
+            0.95, self.event_loop
+        ).rising()
+        self.right_trigger_down_half = self.gamepad.rightTrigger(0.05, self.event_loop)
+        self.right_trigger_up = self.gamepad.rightTrigger(
+            0.95, self.event_loop
+        ).falling()
+
+        # Left trigger events
+        self.left_trigger_down_full = self.gamepad.leftTrigger(
+            0.95, self.event_loop
+        ).rising()
+        self.left_trigger_down_half = self.gamepad.leftTrigger(
+            0.05, self.event_loop
+        ).rising()
+        self.left_trigger_up = self.gamepad.leftTrigger(0.95, self.event_loop).falling()
```

Comment:
nit: These seem completely unused. Should we delete?

Reply:
I think we can delete the trigger events if we are just trying to make the drive base run on joysticks.

Reply:
Good call. We don't really know what sort of control scheme (or HID) the next drive team will prefer, given we haven't finalised the drive team yet.

Reply:
```suggestion
```